### PR TITLE
Facebook login implementation

### PR DESCRIPTION
**Pull request by technowar**

### CODE REVIEW

**Comment by XanderDwyl** [comment]
LTGM

File: utils/config.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"server": {
+		"protocol": "http",
+		"host": "192.168.99.101"
```
Comment (XanderDwyl) [inline utils/config.json]: 192.168.99.100?

**Comment by technowar** [comment]
Please elaborate, thank you.

**Comment by elizar** [comment]
You publicly publish your api key and secret from FB

**Comment by technowar** [comment]
Oh crap na apil commit ang `config.json` haahahaha

**Comment by elizar** [comment]
and please don't use `/facebook` route try `/auth/something` instead

**Pull request closed without merging**